generatedocs/data/params.json

```diff
@@ -141,6 +141,9 @@
     "config.scheduler.settings.concurrentusers": [
         "Number of concurrent users to simulate. Allowed values are positive integers."
     ],
+    "config.scheduler.reconnectsettings" : [
+        "Settings to enable re-connection attempts on unexpected disconnects."
```

Comment:
Suggestion:
"Settings for enabling re-connection attempts in case of unexpected disconnects."